src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -2095,7 +2070,6 @@ module Queries = struct
     ; trust_status
     ; trust_status_all
     ; snark_pool
-    ; blockchain_verification_key
```

Comment:
cc @Schmavery -- we usually don't like backwards incompatible changes to graphql, but since this one was marked experimental, I think it should be okay